prime-router/metadata/schemas/covid-19.schema

```diff
@@ -1260,4 +1260,3 @@ elements:
   - name: sender_id
     type: TEXT
     documentation:  ID name of org that is sending this data to ReportStream.  Suitable for provenance or chain of custody tracking.  Not to be confused with sending_application, in which ReportStream acts as the 'sender' to the downstream jurisdiction.
-
```

Comment:
Why was this line changed? It's not a kotlin file, so it shouldn't be affected by the linter.

Reply:
Leaving it as it was before.

Reply:
Checking master, it doesn't have that character "-"